test/span.js

```diff
@@ -185,31 +184,41 @@ describe('span should', () => {
         assert.isOk(unnormalizedKey in Span._getBaggageHeaderCache());
     });
 
-    describe('with deferred sampling', () => {
```

Comment:
should not be removing this test (it could've been written without explicitly mentioning deferred tag).